beluga/include/beluga/sensor/ndt_sensor_model.hpp

```diff
@@ -165,14 +168,23 @@ class NDTSensorModel {
     };
   }
 
-  /// Returns the L2 likelihood scaled by 'd1' and 'd2' set in the parameters for this instance for 'measurement, or
-  /// 'params_.min_likelihood' if the cell corresponding to 'measurement' doesn't exist in the map.
+  /// Returns the L2 likelihood scaled by 'd1' and 'd2' set in the parameters for this instance for 'measurement', for
+  /// the 3x3 cells around the measurement cell, or 'params_.min_likelihood', whichever is higher.
   [[nodiscard]] double likelihood_at(const ndt_cell_type& measurement) const {
-    const auto maybe_cell = cells_data_.data_near(measurement.mean);
-    if (!maybe_cell.has_value()) {
-      return params_.minimum_likelihood;
+    static const std::array kCellsToConsider{
+        Eigen::Vector2i{-1, -1}, Eigen::Vector2i{-1, -0}, Eigen::Vector2i{-1, 1},
+        Eigen::Vector2i{0, -1},  Eigen::Vector2i{0, 0},   Eigen::Vector2i{0, 1},
+        Eigen::Vector2i{1, -1},  Eigen::Vector2i{1, 0},   Eigen::Vector2i{1, 1},
+    };
```

Comment:
@serraramiro1 meta: some day it would be nice for this kernel to be configurable.

Reply:
Parametrized it but didn't add the configuration point for the ros node as it didn't seem worth it.
We can always add that later.